src/Sylius/Bundle/ShopBundle/Twig/Component/Common/FastestDeliveryTimeComponent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Common;
+
+use Sylius\Bundle\CoreBundle\Provider\DeliveryTimeProviderInterface;
+use Sylius\Bundle\UiBundle\Twig\Component\TemplatePropTrait;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsLiveComponent]
+final class FastestDeliveryTimeComponent
```

Comment:
```suggestion
final class DeliveryTimeComponent
```
there is no need for this prefix, the same in the service and component definitions